docs/prow/prow-jobs.md

```diff
@@ -95,9 +95,11 @@ go run development/tools/cmd/rendertemplates/main.go --config templates/config.y
 In most situations, re-running the job means that Prow uses the same commit. To make sure the job uses the updated code base, you can trigger the job manually.
 Use a tool called [mkpj](https://github.com/kubernetes/test-infra/tree/master/prow/cmd/mkpj) that generates a valid YAML for the Prow job. Follow these steps:
 
-1. Clone the `kubernetes/test-infra` repository. Run:
+1. Clone the `kubernetes/test-infra` repository into `k8s.io` directory. Run:
 
 ```shell
+mkdir -p $GOPATH/src/k8s.io/
+cd $GOPATH/src/k8s.io
 git clone "git@github.com:kubernetes/test-infra.git"
```

Comment:
FYI: you can make it as one-liner:

```shell
git clone "git@github.com:kubernetes/test-infra.git" $GOPATH/src/k8s.io
```

this command will take care of creating the necessary folders

Reply:
nice :)